.github/project_workflows/add_device_profile.yml

```
@@ -0,0 +1,58 @@
+name: Test Add device and regenerate profiles with fastlane match
+
+# SECRETS needed:
+### SSH_PRIVATE_KEY for the match Repo
+### MATCH_PASSWORD
+### APPSTORE_CONNECT_API_KEY
+### API_KEY_ID
+### ISSUER_ID
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      devices:
+        description: 'JSON of devices. `{"Luka iPhone 6": "1234567890123456789012345678901234567890","Felix iPad Air 2": "abcdefghijklmnopqrstvuwxyzabcdefghijklmn"}`'
```

Can shorten the JSON example 😄  
```suggestion
        description: 'JSON of devices. `{"Luka iPhone 6":"1234567890123456789012345678901234567890","Felix iPad Air 2":"abcdefghijklmnopqrstvuwxyzabcdefghijklmn"}`'
```